static/templates/settings/user_groups_admin.hbs

```diff
@@ -1,7 +1,11 @@
 <div id="user-groups-admin" class="settings-section" data-name="user-groups-admin">
     {{#unless is_admin}}
-        {{#if (eq realm_user_group_edit_policy USER_GROUP_EDIT_POLICY_MEMBERS) }}
-        <div class="tip">{{t 'Only group members and organization administrators can modify a group.' }}</div>
+        {{#if (eq realm_user_group_edit_policy policy_values.by_members.code) }}
+        <div class="tip">{{t 'Only group members, organization administrators and moderators can modify a group.' }}</div>
+        {{else if (eq realm_user_group_edit_policy policy_values.by_full_members.code) }}
+        <div class="tip">{{t 'Only full members belonging to the group, organization administrators and moderators can modify a group.' }}</div>
+        {{else if (eq realm_user_group_edit_policy policy_values.by_moderators_only.code) }}
+        <div class="tip">{{t 'Only organization administrators and moderators can modify user groups in this organization.' }}</div>
```

Comment:
I am not super-confident about these messages, any suggestions?

Also, do we still need to show this message to all non-admins users even if they are allowed to modify the user group.

Reply:
@alya for this discussion.  I think it's helpful to explain the security model even if you personally have permission, because you may want to know who else has permission to do so.

Reply:
What is the UI for this?

In general, I find the banners we show when you are *allowed* to do something very distracting/confusing. (E.g. "Anyone in this organization can add bots"). I think we should either figure out a way to make them visually distinct (i.e. not the same yellow banner you see when you *can't* do something), or not show them at all.

Reply:
As of now, we show the yellow banner only, I think we can hide them in case user is allowed as they can also see the setting for knowing what other roles have the permission

Reply:
I think it's probably worth extracting this to a separate issue / #design thread for how to do these tips to indicate permissions where you personally do or don't have the permission.